### PR TITLE
feat(EMS-501): Policy and export - Single contract policy page

### PR DESCRIPTION
**Pull request by ttbarnes**
This PR adds the Single contract policy page to the Policy and export page.

## Summary of changes
- Setup Single contract policy page, route and controller complete with all form fields.
- Update "type of policy" page to redirect to Single/multi contract policy page depending on the submitted policy type.

## Other improvements
- Extract field-ids into directory structure instead of a single file.
- Create shared field-ids for ids that are using in quote and insurance flows.
- Move company house field field strings into it's own directory.
- Add `mockCurrencies` to test-mocks.

### CODE REVIEW

File: e2e-tests/constants/routes/policy-and-export.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ const ROOT = '/policy-and-exports';
 export const POLICY_AND_EXPORTS = {
   ROOT,
   TYPE_OF_POLICY: `${ROOT}/type-of-policy`,
+  SINGLE_CONTRACT_POLICY: `${ROOT}/single-contract-policy`,
```
Comment (Zainzzkk) [inline e2e-tests/constants/routes/policy-and-export.js:6]: I just wonder in the future, is it worth doing quote root and insurance root so we know which is which on quick glance?

File: e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/type-of-policy/type-of-policy.spec.js [hunk 2]
```diff
@@ -212,12 +212,12 @@ context('Insurance - Policy and exports - Type of policy page - As an exporter,
...
 
     describe('when submitting the answer as `single`', () => {
-      it(`should redirect to ${ROUTES.INSURANCE.POLICY_AND_EXPORTS.ABOUT_GOODS_OR_SERVICES}`, () => {
+      it(`should redirect to ${ROUTES.INSURANCE.POLICY_AND_EXPORTS.SINGLE_CONTRACT_POLICY}`, () => {
         singlePolicyField.input().click();
 
         submitButton().click();
 
-        const expected = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.POLICY_AND_EXPORTS.ABOUT_GOODS_OR_SERVICES}`;
+        const expected = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.POLICY_AND_EXPORTS.SINGLE_CONTRACT_POLICY}`;
```
Comment (Zainzzkk) [inline e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/type-of-policy/type-of-policy.spec.js:220]: can we shorten this line?

File: e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/type-of-policy/type-of-policy.spec.js [hunk 3]
```diff
@@ -232,12 +232,12 @@ context('Insurance - Policy and exports - Type of policy page - As an exporter,
...
 
     describe('when submitting the answer as `multiple`', () => {
-      it(`should redirect to ${ROUTES.INSURANCE.POLICY_AND_EXPORTS.ABOUT_GOODS_OR_SERVICES}`, () => {
+      it(`should redirect to ${ROUTES.INSURANCE.POLICY_AND_EXPORTS.MULTI_CONTRACT_POLICY}`, () => {
         multiplePolicyField.input().click();
 
         submitButton().click();
 
-        const expected = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.POLICY_AND_EXPORTS.ABOUT_GOODS_OR_SERVICES}`;
+        const expected = `${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ROUTES.INSURANCE.POLICY_AND_EXPORTS.MULTI_CONTRACT_POLICY}`;
```
Comment (Zainzzkk) [inline e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/type-of-policy/type-of-policy.spec.js:240]: same as above

File: e2e-tests/constants/routes/policy-and-export.js (hunk 1 above)
Comment (ttbarnes) [reply e2e-tests/constants/routes/policy-and-export.js:6]: Good shout - this file should also be in `routes/insurance/type-of-policy` I think, will have a look

**Review by Zainzzkk — APPROVED** [review]
lgtm

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**